3rdParty/fuerte/src/HttpConnection.cpp

```diff
@@ -305,7 +314,8 @@ std::string HttpConnection<ST>::buildRequestBody(Request const& req) {
     header.append("Connection: Close\r\n");
   }
 
-  if (req.contentType() != ContentType::Custom) {
+  if (req.header.restVerb != RestVerb::Get &&
```

Comment:
This may be correct, but it is not clear to me why we don't send a Content-Type for HTTP GET.
Can we add a comment here for why this is necessary?

Reply:
Another unrelated comment regarding one of the files in this PR:
it seems that in https://github.com/arangodb/arangodb/pull/10268/files#diff-c4c73f0cf79f341d3fddc85d9d0bae19R300 we always copy the headers. We should change the for by-value-for iteration to a by-reference-for iteration to avoid the copying!

Reply:
because GET has no body content ?

Reply:
which line do you mean? VPackObjectIterator always returns a temporary

Reply:
This line (please click on link to see it): https://github.com/arangodb/arangodb/pull/10268/files#diff-c4c73f0cf79f341d3fddc85d9d0bae19R300

But I fixed this already in a PR of mine, so it can be ignored here.